cabinets/cabinet/__init__.py

```diff
@@ -122,6 +122,18 @@ def delete(cls, path: str, **kwargs):
         """
         cls.delete_content(path, **kwargs)
 
+    @classmethod
+    def list(cls, directory: str, **kwargs) -> List[str]:
```

Comment:
Are `Path` objects acceptable here? Like do we still want `Union[str, Path]`

Reply:
Yes